slack_bot.js

```diff
@@ -342,7 +342,7 @@ controller.hears(["^set role for <@(.*)> to (.*)"], 'direct_mention,direct_messa
 });
 
 function hasApprovedEmailDomain(email) {
-  if (email.match(".*\.gov\.uk$")) {
+  if (email.match(".*(\.gov|naturalengland\.org){1}\.uk$")) {
```

Comment:
This will accept bob@idontworkfornaturalengland.org.uk — should be `.*\.(gov|naturalengland\.org)\.uk$` or something